Sources/Rendering/Core/VolumeProperty/index.d.ts

```diff
@@ -204,11 +204,20 @@ export interface vtkVolumeProperty extends vtkObject {
 	 */
 	setIndependentComponents(independentComponents: boolean): boolean;
 
-  /**
-   *
-   * @param {Number | Number[]} labelOutlineThickness
-  */
-  setLabelOutlineThickness(labelOutlineThickness: number | number[]): boolean;
+	/**
+	 * It will set the label outline thickness for the labelmaps. It can accept
+	 * a single number or an array of numbers. If a single number is provided,
+	 * it will be used for all the components. If an array is provided, it indicates
+	 * the thickness for each segment index. For instance if you have a labelmap
+	 * with 3 segments (0: background 1: live 2: tumor), you can set the thickness
+	 * to [2,4] to have a thicker outline for the tumor (thickness 4). It should be 
+	 * noted that if the number of segments is greater than the number of thickness
+	 * the first thickness will be used for the remaining segments.
```

Comment:
how about having the first array value control the "default thickness" ?

Your condition `if the number of segments is greater than the number of thickness the first thickness will be used for the remaining segments` would otherwise be an issue if you want to "emphasize" the first label (because it is the current label) without impacting the other labels that rely on the default value.

I also wonder if the label of value 0 is ALWAYS background hence not visible. Maybe there could be usefulness to have an actual representation (therefore thickness) for the background value (label=0)

Reply:
I updated the doc. 
I'm not entirely certain about the background thickness, as it can be a bit challenging around the edges, I suppose. I am not modifying the outline thickness behaviour for background in this PR, so perhaps we could consider waiting for that specific scenario and create a separate PR for adjusting the background's outline thickness?